src/main/scala/com/codacy/client/bitbucket/client/BitbucketClient.scala

```diff
@@ -78,6 +78,14 @@ class BitbucketClient(key: String, secretKey: String, token: String, secretToken
     value
   }
 
+  def postForm[T](request: Request[T], values: Map[String, Seq[String]])(implicit reader: Reads[T]): RequestResponse[T] = {
```

Comment:
The names look really good. If you look at other libraries with same
functionality they also use postForm. Play WS is not a good example in this
case. Scala http is.

I really prefer stuff to be explicit. Why leave stuff for interpretation
when you can make it explicit.
On Tue, 1 Dec 2015 at 9:12 AM, johannegger <notifications@github.com> wrote:

> In src/main/scala/com/codacy/client/bitbucket/client/BitbucketClient.scala
> <https://github.com/codacy/bitbucket-scala-client/pull/9#discussion_r46254539>
> :
>
> > @@ -78,6 +78,14 @@ class BitbucketClient(key: String, secretKey: String, token: String, secretToken
> >      value
> >    }
> >
> > +  def postForm[T](request: Request[T], values: Map[String, Seq[String]])(implicit reader: Reads[T]): RequestResponse[T] = {
>
> i already talked to pedro. i just don't like the names postForm and
> postJson. i don't mind hiding the general method but i'd really prefer if
> the other two were both named post with different signatures
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codacy/bitbucket-scala-client/pull/9/files#r46254539>.
>